test/sql/storage/tpch/tpch_storage_bitpacking.test_slow

```diff
@@ -0,0 +1,157 @@
+# name: test/sql/storage/tpch/tpch_storage_bitpacking.test_slow
```

Comment:
Test requests:

* Could you add an exhaustive tests that runs bitpacking on a table with values that fit in each of the different bit-widths (1 bit, 2 bits, 3 bits, .., 63 bits)?
* The same test but with negative values (e.g. 1000x`-1`, 1000x`-2`, 1000x`-4`, etc)
* Tests with lists, particularly many lists with odd/varying sizes, and tests involving having a filter on another column so that certain lists are skipped. This might find some interesting edge cases since in this scenario `Scan` and `Skip` will be called with non-vector aligned values.

Reply:
Added the tests covering all bit widths! It did indeed find a problem with the bitpacking width debug assertion that previously went unnoticed!

Also added the same list tests that are also run for RLE, which i think cover the cases you describe here. must have missed them!